docs/concepts/storage/volumes.md

```diff
@@ -507,68 +507,70 @@ See the [iSCSI example](https://github.com/kubernetes/examples/tree/{{page.githu
 
 ### local
 
-{% assign for_k8s_version="v1.7" %}{% include feature-state-alpha.md %}
-
-This alpha feature requires the `PersistentLocalVolumes` feature gate to be
-enabled.
-
-**Note:** Starting in 1.9, the `VolumeScheduling` feature gate must also be enabled.
-{: .note}
```

Comment:
Please confirm that both feature gates are now enabled by default so users don't need to take extra care.

Reply:
yes